internal/btf/core.go

```diff
@@ -509,8 +549,10 @@ func (ca coreAccessor) enumValue(t Type) (*EnumValue, error) {
 }
 
 type coreField struct {
-	Type   Type
-	offset uint32
+	Type           Type
+	offset         uint32
```

Comment:
Maybe the `offset` field should be renamed `bitOffset` to better know its unit (bit vs. byte).

Reply:
not my code originally, but I agree.